plugin/core/context.py

```
@@ -66,6 +210,126 @@ def push_message(
+            except Exception:
+                pass
+
+        if zmq_client is not None:
+            try:
+                from plugin.settings import (
+                    PLUGIN_ZMQ_MESSAGE_PUSH_SYNC_TIMEOUT,
+                    PLUGIN_ZMQ_MESSAGE_PUSH_ENDPOINT,
+                    PLUGIN_ZMQ_MESSAGE_PUSH_BATCH_SIZE,
+                    PLUGIN_ZMQ_MESSAGE_PUSH_FLUSH_INTERVAL_MS,
+                )
+            except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)